<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-QE6D7HJ" ignored="0" name="Default test" tests="6" failures="0" timestamp="25 abr 2018 14:00:39 GMT" time="0.122" errors="0">
  <testcase name="@BeforeClass setUp" time="0.016" classname="utils.baseTest">
    <failure type="java.lang.Error" message="Unresolved compilation problem: 
	Unreachable code
">
      <![CDATA[java.lang.Error: Unresolved compilation problem: Unreachable codeat utils.baseTest.getDriver(baseTest.java:40)at utils.baseTest.setUp(baseTest.java:22)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)at java.lang.reflect.Method.invoke(Unknown Source)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:451)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:163)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
    </failure>
  </testcase> <!-- @BeforeClass setUp -->
  <testcase name="@AfterClass tearDown" time="0.0" classname="utils.baseTest">
    <skipped/>
  </testcase> <!-- @AfterClass tearDown -->
  <testcase name="testHomePage" time="0.0" classname="tests.testHome">
    <skipped/>
  </testcase> <!-- testHomePage -->
  <testcase name="testListaPresente" time="0.0" classname="tests.testHome">
    <skipped/>
  </testcase> <!-- testListaPresente -->
  <testcase name="testNossasLojas" time="0.0" classname="tests.testHome">
    <skipped/>
  </testcase> <!-- testNossasLojas -->
  <testcase name="testLogin" time="0.0" classname="tests.testHome">
    <skipped/>
  </testcase> <!-- testLogin -->
  <testcase name="testCarrinho" time="0.0" classname="tests.testHome">
    <skipped/>
  </testcase> <!-- testCarrinho -->
  <testcase name="testLogOut" time="0.0" classname="tests.testHome">
    <skipped/>
  </testcase> <!-- testLogOut -->
</testsuite> <!-- Default test -->
